backend/data/models.json

```diff
@@ -0,0 +1,127 @@
+[
```

Comment:
Why make this a list instead of a dict. 
(Then we can directly lookup the id instead of needing to search for it)